### PR TITLE
Sprint 2/issue 12#login screen

### PR DESCRIPTION
**Pull request by J3fte**
# Description

- The login screen was implemented, which validates that the inputs are not empty, and also validates that an email must be entered

image

### CODE REVIEW

**Review by Axw3ll — APPROVED** [review]

File: renta_facil_colima/package.json [hunk 1]
```diff
@@ -15,10 +15,14 @@
...
     "react": "18.2.0",
     "react-native": "0.72.3",
+<<<<<<< HEAD
+    "react-native-drop-shadow": "^0.0.7",
+=======
     "react-native-gesture-handler": "^2.12.1",
     "react-native-paper": "4.12.5",
     "react-native-safe-area-context": "^4.7.1",
     "react-native-screens": "^3.24.0",
+>>>>>>> f6f77a34ea1c3a070e4775f734d4d4d5b8f9c738
```
Comment (BrandonVG) [inline renta_facil_colima/package.json:25]: You forgot to resolve those merge conflict

**Review by Tole15 — APPROVED** [review]
I'm glad to hear that the code is working for you! I strongly recommend using React Hook Forms to handle all the logic in the inputs. It will help simplify your code and enhance the user experience. It would also help in the error management.

**Review by BrandonVG — CHANGES_REQUESTED** [review]

File: renta_facil_colima/package.json (hunk 1 above)
Comment (BrandonVG) [inline renta_facil_colima/package.json]: You forgot to delete those merge conflict

File: renta_facil_colima/yarn.lock [hunk 2]
```diff
@@ -5533,6 +5533,16 @@ react-is@^17.0.1:
...
+<<<<<<< HEAD
+react-native-drop-shadow@^0.0.7:
+  version "0.0.7"
+  resolved "https://registry.npmjs.org/react-native-drop-shadow/-/react-native-drop-shadow-0.0.7.tgz"
+  integrity sha512-TsY0RpikcU/TwtMy3vyYEXxCDK6zdlptw8zZqoV4SASCo0tTsZSdq7fbQF1dsH0/49w7ts7CYEOTbqT6InecfQ==
+  dependencies:
+    logkitty "^0.7.1"
+
+react-native-vector-icons@^10.0.0:
+=======
```
Comment (BrandonVG) [inline renta_facil_colima/yarn.lock]: You forgot to delete those merge conflict

File: renta_facil_colima/yarn.lock [hunk 3]
```diff
@@ -5572,14 +5582,19 @@ react-native-screens@^3.24.0, "react-native-screens@>= 3.0.0":
...
   integrity sha512-efMOVbZIebY8RszZPzPBoXi9pvD/NFYmjIDYxRoc9LYSzV8rMJtT8FfcO2hPu85Rn2x9xktha0+qn0B7EqMAcQ==
   dependencies:
     prop-types "^15.7.2"
     yargs "^16.1.1"
 
+<<<<<<< HEAD
+react-native@*, react-native@>=0.61.5, react-native@0.72.3:
+=======
 react-native@*, react-native@>=0.42.0, react-native@0.72.3:
+>>>>>>> f6f77a34ea1c3a070e4775f734d4d4d5b8f9c738
```
Comment (BrandonVG) [inline renta_facil_colima/yarn.lock]: You forgot to delete those merge conflict

File: renta_facil_colima/src/styles/GoogleButtonStyle.js [hunk 4]
```diff
@@ -15,7 +15,7 @@ const GoogleButtonStyles = StyleSheet.create({
   },
   text: {
     marginLeft: 10,
-    fontSize: 24,
+    fontSize: 20,
```
Comment (BrandonVG) [inline renta_facil_colima/src/styles/GoogleButtonStyle.js]: The fontsize in the figma design is 24 not 20, this change is unnecesary

**Review by BrandonVG — CHANGES_REQUESTED** [review]

**Review by BrandonVG — APPROVED** [review]

**Pull request merged**